handsontable/src/plugins/filters/__tests__/filtersUI.spec.js

```diff
@@ -1058,6 +1058,37 @@ describe('Filters UI', () => {
       expect(byValueMultipleSelect().element.querySelector('.htCore td').textContent).toBe('(Blank cells)');
     });
 
+    it('should utilize the `displayValue` cell meta property to display the cell value', () => {
+      const columnsSetting = getColumnsForFilters();
+
+      columnsSetting[1].renderer = (instance, td, row, col, prop, value, cellProperties) => {
+        cellProperties.displayValue = `Custom ${value}`;
+
+        Handsontable.renderers.TextRenderer.apply(this, [instance, td, row, col, prop, value, cellProperties]);
+      };
+
+      handsontable({
+        data: getDataForFilters(),
+        columns: columnsSetting,
+        filters: true,
+        dropdownMenu: true,
+        width: 500,
+        height: 300
+      });
+
+      dropdownMenu(1);
+
+      const unifiedColDataSample = [
+        'Alice Blake', 'Alyssa Francis', 'Becky Ross', 'Bridges Sawyer', 'Burt Cash', 'Carissa Villarreal'
+      ];
+
+      for (let i = 0; i < unifiedColDataSample.length; i++) {
+        expect(
+          byValueMultipleSelect().element.querySelectorAll('.htCore td')[i].textContent
+        ).toBe(`Custom ${unifiedColDataSample[i]}`);
+      }
```

Comment:
```suggestion
      for (let i = 0; i < unifiedColDataSample.length; i++) {
        expect(
          byValueMultipleSelect().element.querySelectorAll('.htCore td')[i].textContent
        ).toBe(`Custom ${unifiedColDataSample[i]}`);
      }
      expect(unifiedColDataSample.length).toBe(6);
```
I'd secure that `for` with one more `expect`.

Reply:
I assume that there is a chance a developer (unintentionally, by accident) can edit `for` loop or edit an array in a way that there won't be any iteration. Then the test will pass, as no `expect` is called.